openpathsampling/tools.py

```diff
@@ -205,24 +205,20 @@ def units_from_snapshot(snapshot):
         the simtk.unit.Unit to be used
     """
 
-    units = {}
+    units = {'velocity': None, 'energy': None, 'length': None}
```

Comment:
Whoops, just saw the note in #384 that this is coming in a later PR (I assume that's #396?). Nevermind.